src/H5Dfarray.c

```diff
@@ -1435,7 +1435,7 @@ H5D__farray_idx_copy_setup(const H5D_chk_idx_info_t *idx_info_src, const H5D_chk
             HGOTO_ERROR(H5E_DATASET, H5E_CANTOPENOBJ, FAIL, "can't open fixed array")
 
     /* Set copied metadata tag */
-    H5_BEGIN_TAG(H5AC__COPIED_TAG);
+    H5_BEGIN_TAG(H5AC__COPIED_TAG)
```

Comment:
I don't like macros that float in space like they aren't C statements. We should fix up the macros so they are C-like when used.

Reply:
>I don't like macros that float in space like they aren't C statements.

Meaning you'd prefer them with no indentation?

>We should fix up the macros so they are C-like when used.

C-like?  You mean function-like?  I don't see how.  Recall the macro is:

```C
#define H5_BEGIN_TAG(tag)                    \
    {                                        \
        haddr_t prv_tag = HADDR_UNDEF;       \
        H5AC_tag(tag, &prv_tag);
```

It's not function-like at all, it declares a variable that survives in scope.  As it's not function-like, it's better without the trailing semi, IMHO.